### PR TITLE
Add information on how to possibly fix database connection errors.

### PR DESCRIPTION
**Pull request by MatheusRV**
> ⚠️ We have stopped merging PRs for now to the Strapi core.<br><br>
> The reason is that we are developing new architecture for the admin panel and for the plugins.<br>
> This new architecture will provide stability of the Strapi core as we approach the release of Beta.<br>
> We appreciate and welcome all your contributions, but until further notice, please do not submit a PR as it will not be merged.<br>
> Furthermore, you will have to rewrite it based on the new architecture.

#### My PR is a:
- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix #issueNumber
- [X] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:
- [ ] Admin
- [ ] Documentation
- [X] Framework
- [ ] Plugin

#### Manual testing done on the following databases:
- [X] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: packages/strapi-hook-bookshelf/lib/utils/connectivity.js [hunk 1]
```diff
@@ -85,7 +85,7 @@ module.exports = (scope, success, error) => {
       if (err.sql) {
         console.log('⚠️  Server connection has failed! Make sure your database server is running.');
       } else {
-        console.log(`⚠️  Database connection has failed! Make sure your "${scope.database.settings.database}" database exist.`);
+        console.log(`⚠️  Database connection has failed! Make sure your "${scope.database.settings.database}" database exist or user "${scope.database.settings.database.user}" have all privileges.`);
```
Comment (MatheusRV) [inline packages/strapi-hook-bookshelf/lib/utils/connectivity.js]: Am I correctly referencing the user?

**Comment by lauriejim** [comment]
Hello! Can you give us the reason of this change please.
What is the situation you faced and that change can help you.

**Comment by derrickmehaffy** [comment]
@MatheusRV I'm not sure I see the point in referencing the user in the database connection error?

Permissions are permissions, the first means to check for them is to use a local client (or a CLI client) to test the connection if that fails then your user/permissions are wrong. If not, then there is a problem either with how they are being sent to strapi (env or whatever) or a problem with Strapi itself.

I don't see any benefit to logging the user to the console, and could honestly contribute to being a security risk since on linux all console messages are logged on the system.

**Comment by MatheusRV** [comment]
I was trying to install strapi on a mysql database and I had the error message that maybe the bank had not been created and several times I checked to see if the bank really was created. So I decided to give the bank permission to the user so everything worked.

**Comment by lauriejim** [comment]
I close this PR, in the beta the error message is displayed.

**Comment by lauriejim** [comment]
Thank you for your PR

**Pull request closed without merging**